module/VuFind/src/VuFind/ILS/Driver/PAIA.php

```diff
@@ -224,6 +232,12 @@ public function init()
         } else {
             $this->debug('Caching not enabled, disabling it by default.');
         }
+
+        if (isset($this->config['PAIA']['muteAccountBlocksNotificationForScopes'])) {
```

Comment:
This could be reduced to:

`$this->mutedScopes = $this->config['PAIA']['muteAccountBlocksNotificationForScopes'] ?? [];`